README.md

```diff
@@ -286,6 +286,16 @@ You can find an example of an upper constraints file
 [here](http://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt?h=stable/pike).
 In case the `<version_number>` is omitted, the newest one will be used.
 
+`<version_number>` could be standard SEMVER version number that is available
+on <https://pypi.org/project/monasca-ui/#history>. It also support creating
+installer straight from monasca-ui Git repository. For this you need
+to provide commit hash (e.g. `d58f0f6`), branch (e.g. `stable/pike`) or tag.
+
+You could create installer from custom fork of monasca-ui. Just set
+`MONASCA_UI_GIT_REPO` variable to your fork Git URI.
```

Comment:
I think git repository should be set by script parameter not system variable.

Reply:
We have discussed that and decided to use shell variable for the sake of simplicity. Any option would work for me though.